packages/api-headless-cms/src/types.ts

```diff
@@ -2136,6 +2136,7 @@ export interface OnEntryPublishErrorTopicParams {
 export interface OnEntryBeforeRepublishTopicParams {
     entry: CmsEntry;
     model: CmsModel;
+    storageEntry: CmsEntry;
```

Comment:
I would prefer not to expose storage layer objects to 3rd party developers. If this is not necessary, let's not add this. Also, if you know other places that expose this, let's remove it from those places.